vembrane/ann_types.py

```diff
@@ -414,8 +448,8 @@ def convert(self, key: str, value: str) -> Tuple[str, AnnotationType]:
     "INTRON": AnnotationNumberTotalEntry(
         "INTRON", description="The intron number (out of total number)"
     ),
-    "EXON": AnnotationNumberTotalEntry(
-        "EXON", description="The exon number (out of total number)"
+    "EXON": AnnotationRangeTotalEntry(
+        "EXON", description="The exon position range (out of total number of exons)"
```

Comment:
I don't quite understand if this range really gives a location estimate in bp or if it gives a range of (enumerated) exons, so perhaps we should use "index" instead of "position" if the latter is the case?
```suggestion
"EXON", description="The exon index range (out of total number of exons)"
```